db/src/androidTest/java/io/lantern/db/DBTest.kt

```diff
@@ -197,7 +197,7 @@ class DBTest {
                     PathAndValue("/messages/c", Message("Message C blah blah")),
                     PathAndValue("/messages/d", Message("Message D blah blah blah"))
                 ),
-                db.list<String>("/messages/%")
+                db.list<Message>("/messages/%")
```

Comment:
Drive-by, this test was using the wrong type, but it didn't break anything because we never tried accessing anything from that type.